SmartSyncExplorerReactNative/installandroid.js

```diff
@@ -8,7 +8,7 @@ console.log('Installing npm dependencies');
 execSync('npm install', {stdio:[0,1,2]});
 var rimraf = require('rimraf');
 console.log('Installing sdk dependencies');
-var sdkDependency = 'salesforcemobilesdk-android';
+var sdkDependency = 'SalesforceMobileSDK-Android';
```

Comment:
This was the only template using lowercase and it was causing path issues while copying resources. I lined it up to be consistent with the other templates.